common/djangoapps/student/urls.py

```diff
@@ -45,6 +45,7 @@
 if settings.FEATURES.get('AUTOMATIC_AUTH_FOR_TESTING'):
     urlpatterns += (
         url(r'^auto_auth$', 'auto_auth'),
+        url(r'^update_user$', 'update_user'),
```

Comment:
it will not map to existing frontend but test method will call this function to update a user's status